packages/fiori/src/IllustratedMessage.js

```diff
@@ -306,6 +306,7 @@ class IllustratedMessage extends UI5Element {
 		super();
 
 		this._handleResize = this.handleResize.bind(this);
+		this._lastKnownBCRForMedia = {};
```

Comment:
Comment what the object will store would be good, maybe known abr. but I could not recall "bcr"